lib/ReactViews/Map/Panels/InnerPanel.jsx

```diff
@@ -16,6 +16,8 @@ const InnerPanel = createReactClass({
      * panel to close.
      */
     doNotCloseFlag: PropTypes.string,
+    /** Disable closing on loss of focus and only allow with close button */
+    disableCloseOnFocusLoss: PropTypes.boolean,
```

Comment:
I think this should be `PropTypes.bool` instead. It might be causing this warning in the console:

```
Warning: Failed prop type: InnerPanel: prop type `disableCloseOnFocusLoss` is invalid; it must be a function, usually from the `prop-types` package, but received `undefined`.This often happens because of typos such as `PropTypes.function` instead of `PropTypes.func`.
```

Reply:
Resolved, yes that error no longer occurs with your change thanks!